CRToast/CRToastView.m

```diff
@@ -131,7 +131,7 @@ - (void)layoutSubviews {
     CGSize imageSize = self.imageView.image.size;
     CGFloat preferredPadding = self.toast.preferredPadding;
     
-    CGFloat statusBarYOffset = self.toast.displayUnderStatusBar ? (CRGetStatusBarHeight()+CRStatusBarViewUnderStatusBarYOffsetAdjustment) : 0;
+    CGFloat statusBarYOffset = MAX(self.toast.displayUnderStatusBar ? (CRGetStatusBarHeight()+CRStatusBarViewUnderStatusBarYOffsetAdjustment) : 0, 0);
```

Comment:
Is there a situation where this was going negative? I only ask because I feel that the `MAX` call would only be used to resolve some kind of bug/issue.I feel like this going below 0 is its own problem we should probably address. Do you have details on the circumstances this happened under?